### ISSUE TITLE
"Unused import" when an alias is actually used

### ISSUE THREAD

**Issue by rehmsen** [palantir/tslint#1566]
### Bug Report

- __TSLint version__: `3.15.1`
- __TypeScript version__: `1.9.0`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
declare module 'foo:bar' {
  import bar = baz.bar;
  export import Qux = bar.Qux;
  ...
}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "array-type": [true, "array"],
    "class-name": true,
    "forin": true,
    "interface-name": [true, "never-prefix"],
    "jsdoc-format": true,
    "label-position": true,
    "label-undefined": true,
    "new-parens": true,
    "no-angle-bracket-type-assertion": true,
    "no-construct": true,
    "no-debugger": true,
    "no-namespace": [true, "allow-declarations"],
    "no-reference": true,
    "no-require-imports": true,
    "no-unused-expression": true,
    "no-unused-variable": true,
    "no-use-before-declare": true,
    "no-var-keyword": true,
    "semicolon": [true, "always"],
    "switch-default": true,
    "triple-equals": [true, "allow-null-check"],
    "use-isnan": true,
    "variable-name": [
      true,
      "check-format",
      "ban-keywords",
      "allow-leading-underscore",
      "allow-trailing-underscore"
    ]
  },
  "rulesDirectory": [
    "rules"
  ]
}
```

#### Actual behavior

Unused import: 'bar'

#### Expected behavior

No error - bar is not unused, it is used in the next line.

**Comment by adidahiya** [comment]
this is likely an issue in the typescript language services. any chance you could try upgrading to TS 2.0 and see if it still repros then?

related: #684, #325

**Comment by rehmsen** [comment]
We are in the progress of updating to Typescript 2.0, but this out of my control and might take a few more days. 
I created a small repo project and it indeed does not happen with Typescript 2.0 - then I tried to downgrade Typescript to 1.9 in the demo project (and uninstalling the global tsc), but I still cannot reproduce:

    $ node_modules/.bin/tsc --version
    Version 1.9.0-dev.20160627-1.0
    $ node_modules/.bin/tslint --version
    3.15.1
    $ node_modules/.bin/tslint bar.d.ts  // returns no errors

I am using the same `tslint.json` except for the custom rules, so I am a bit stumped. I think I might wait a few days until our real repo is hopefully on Typescript 2, and then report back.

**Comment by rehmsen** [comment]
Okay, we are on Typescript 2 now and this is fixed. Thanks a lot, closing.
